README.md

```diff
@@ -26,12 +26,22 @@ To build a distribution:
 ```
 (check `build/distributions/` then).
 
-Integration tests can be run with 
+Integration tests can be run with
 ```bash
 ./gradlew integrationTest
 ```
 and require [Docker](https://www.docker.com/) installed.
 
+# Formatting
+
+Checkstyle is used for verifying code formatting.
+For misc files Spotless is used.
```

Comment:
How to read this?

Checkstyle verifies code formatting. Spotless verifies others. But `spotlessApply` formats Java? Why couldn't Spotless also verify Java?

Reply:
No, like written in. Spotless formats misc files.

Reply:
OK.... But still I think Checkstyle could be removed and Spotless could both format and verify also code formatting.